.github/workflows/test.yml

```diff
@@ -62,6 +62,9 @@ jobs:
           test_format: JUNIT
           test_framework: JEST
           test_path: ./coverage/junit.xml
+          coverage_format: LCOV/TXT
```

Comment:
This format is not listed in [runforesight/foresight-test-kit-action's docs](https://github.com/runforesight/foresight-test-kit-action#available-coverage-formats)

Reply:
https://github.com/runforesight/foresight-test-kit-action/pull/10 added this one to the GH readme. We had already mentioned on our Docs page (https://docs.runforesight.com/advanced-configuration/test-kit#available-coverage-formats)